### PR TITLE
Log in

### PR DESCRIPTION
**Pull request by tamarailic**

### CODE REVIEW

File: APP/ev_chargers/lib/models/user.dart [hunk 1]
```diff
@@ -116,4 +115,27 @@ class User {
...
+
+  static Future<String> logIn(String email, String password) async {
+    bool trustSelfSigned = true;
+    HttpClient httpClient = HttpClient()
+      ..badCertificateCallback =
+          ((X509Certificate cert, String host, int port) => trustSelfSigned);
+    IOClient ioClient = IOClient(httpClient);
+
+    var response = await ioClient.get(
+      Uri.parse('$url/User/login?id=' + email + "&newPassword=" + password),
```
Comment (UPocek) [inline APP/ev_chargers/lib/models/user.dart:127]: Why newPassword?

File: APP/ev_chargers/lib/models/user.dart [hunk 2]
```diff
@@ -116,4 +115,27 @@ class User {
...
+      headers: {
+        HttpHeaders.contentTypeHeader: 'application/json',
+      },
+    );
+    if (response.statusCode == 200) {
+      var userData = jsonDecode(response.body);
+      user = User(userData['id'], userData['firstName'], userData['lastName'],
+          userData['email'], userData['password'], 0.0);
```
Comment (UPocek) [inline APP/ev_chargers/lib/models/user.dart:135]: Don't use 0.0 this is used only when new user is created use userData['accountBalance'] instead.

File: APP/ev_chargers/lib/models/user.dart [hunk 3]
```diff
@@ -116,4 +115,27 @@ class User {
...
+      return jsonDecode(response.body)["id"];
```
Comment (UPocek) [inline APP/ev_chargers/lib/models/user.dart:136]: Are you sure you need to return this?

Comment (UPocek) [reply APP/ev_chargers/lib/models/user.dart:136]: Maybe return just bool instead

File: APP/ev_chargers/lib/screens/login_screen.dart [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                      child: const Text("Sign in"),
+                    ),
+                    Padding(
+                        padding: const EdgeInsets.only(top: 10.0, bottom: 10.0),
+                        child: InkWell(
+                            child: const Text("Don't have an account yet?"),
+                            onTap: () {
+                              Navigator.of(context).push(MaterialPageRoute(
+                                  builder: ((context) =>
+                                      RegistrationScreen())));
```
Comment (UPocek) [inline APP/ev_chargers/lib/screens/login_screen.dart:57]: Fix formatting by adding ,

**Review by UPocek — APPROVED** [review]

File: APP/ev_chargers/lib/models/user.dart (hunk 1 above)
Comment (tamarailic) [reply APP/ev_chargers/lib/models/user.dart:127]: Wrong place, meant it in other method.

File: APP/ev_chargers/lib/models/user.dart (hunk 2 above)
Comment (tamarailic) [reply APP/ev_chargers/lib/models/user.dart:135]: I didn't use it, it was there when i pulled it.

**Pull request merged**